### PR TITLE
fix(suite-native): do not display fiat amount for testnet coins

### PR DESCRIPTION
**Pull request by yanascz**
## Screenshots:
image
image
image

### CODE REVIEW

File: suite-native/formatters/src/components/CryptoToFiatAmountFormatter.tsx [hunk 1]
```diff
@@ -34,7 +29,5 @@ export const CryptoToFiatAmountFormatter = ({
         cryptoValue: value,
     });
 
-    const formattedFiatValue = FiatAmountFormatter.format(fiatValue ?? '0');
-
-    return <AmountText value={formattedFiatValue} isDiscreetText={isDiscreetText} {...textProps} />;
+    return <FiatAmountFormatter network={network} value={fiatValue} {...textProps} />;
```
Comment (vytick) [inline suite-native/formatters/src/components/CryptoToFiatAmountFormatter.tsx:32]: why to exclude `isDiscreetText`?

Comment (yanascz) [reply suite-native/formatters/src/components/CryptoToFiatAmountFormatter.tsx:32]: It's not used anywhere with this component and the default is the same as in `FiatAmountFormatter`. It's practically dead code. ☠️

**Review by PeKne — APPROVED** [review]

File: suite-native/formatters/src/components/CryptoToFiatAmountFormatter.tsx (hunk 1 above)
Comment (vytick) [reply suite-native/formatters/src/components/CryptoToFiatAmountFormatter.tsx:32]: makes sense 👍

**Comment by STew790** [comment]
**QA OK**
I don't see fiat value anywhere.

**Info:**
24.9.1 https://github.com/trezor/trezor-suite/commit/306d448b5e568d81359228294e847c80cea6c25e

**Pull request merged**